data_collection/gazette/spiders/pa_cameta.py

```diff
@@ -0,0 +1,80 @@
+import datetime
+import re
+
+import scrapy
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+RE_EDITION_NUMBER = re.compile(r"Edi..o\s*(\d+)")
+RE_EXT_DATE = re.compile(r"\s+(\d+)\sde\s(\w+)\sde\s(\d+)")
+
+
+class PaCameta(BaseGazetteSpider):
+    TERRITORY_ID = "1502103"
+    name = "pa_cameta"
+    allowed_domains = ["prefeituradecameta.pa.gov.br"]
+    start_date = datetime.date(2023, 1, 12)
+
+    BASE_URL = "https://prefeituradecameta.pa.gov.br/diario-oficial-do-municipio/"
+
+    def start_requests(self):
+        url = f"{self.BASE_URL}"
+        yield scrapy.Request(url, callback=self.parse)
+
+    def parse(self, response):
+        a_elements = response.css("#post-20884 > div > ul > li > a")
```

Comment:
```suggestion
        a_elements = response.css("article li > a")
```
[modificação] Para simplificar um pouco o seletor, mas também para torná-lo menos rígido caso a hierarquia mude.